meilisearch-http/build.rs

```diff
@@ -7,4 +17,65 @@ fn main() {
 
     // Generate the 'cargo:' key output
     generate_cargo_keys(ConstantsFlags::all()).expect("Unable to generate the cargo keys!");
+
+    if env::var("CARGO_FEATURE_MINI_DASHBOARD").is_ok() {
+        setup_mini_dashboard().expect("Could not load the mini-dashboard assets");
+    }
+}
+
+fn setup_mini_dashboard() -> anyhow::Result<()> {
+    let cargo_manifest_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+    let cargo_toml = cargo_manifest_dir.join("Cargo.toml");
+
+    let sha1_path = cargo_manifest_dir.join(".mini-dashboard.sha1");
```

Comment:
```suggestion
    let sha1_path = cargo_manifest_dir.join("mini-dashboard.sha1");
```

I am not sure why it should be hidden.